popmon/visualization/templates/card.html

```diff
@@ -1,16 +1,37 @@
-<div class="col-md-{% if 'full_width' in metric %}12{% else %}6{% endif %} mb-5">
+<div class="col-md-{% if 'full_width' in metric %}12{% else %}6{% endif %} mb-5" >
     <a name="{% if feature%}{{ feature.name }}-{%endif%}{{ metric.name }}"></a>
     <div class="card shadow-sm">
-        <div class="card-body" style="text-align: center">
+        <div class="card-body" id="{% if feature%}{{ feature.name }}-{%endif%}{{ metric.name }}-card" style="text-align: center">
             <h4 class="card-title">{{metric.name | fmt_metric}}</h4>
             {% if metric.description|length %}
                 <p class="card-text">{{metric.description}}</p>
             {% endif %}
         </div>
-        {% if 'table' in metric.plot %}
+        <div id= "{% if feature%}{{ feature.name }}-{%endif%}{{ metric.name }}"> </div>
+        {% if (('Overview' in section_title) or ('traffic_light' in metric.type)) %}
             {{ metric.plot }}
         {% else %}
-        <img class="card-img-top" src="data:image/png;base64,{{metric.plot}}" alt="" />
-        {% endif %}
+            <div>
+                <span class="skeleton-loader" id="{% if feature%}{{ feature.name }}-{%endif%}{{ metric.name }}-loading"></span>
+            </div>
+            <script>
+                var feature{{ section_index }}{{ curr }}{{ plt }}_rendered = false ;
+                var observer = new IntersectionObserver(function(entries) {
+                    if(entries[0].isIntersecting === true && feature{{ section_index }}{{ curr }}{{ plt }}_rendered === false ){
+                        plot_container_element = document.getElementById("{% if feature%}{{ feature.name }}-{%endif%}{{ metric.name }}");
+                        var layout = JSON.parse(JSON.stringify(feature{{ section_index }}{{ curr }}_layout["{{ metric.type }}"])) ;
+                        {% if metric.shapes | length %}
+                            layout["shapes"] = {{ metric.shapes | tojson }} ;
+                        {% endif %}
+                        {% if metric.yaxis_range | length %}
+                            layout["yaxis"]["range"] = {{ metric.yaxis_range }} ;
+                        {% endif %}
+                        Plotly.newPlot(plot_container_element, JSON.parse('{{ metric.plot | tojson }}'), layout,{scrollZoom: true, displaylogo: false, modeBarButtonsToRemove: ['lasso2d']}).then(function() { document.getElementById("{% if feature%}{{ feature.name }}-{%endif%}{{ metric.name }}-loading").remove(); });                             
```

Comment:
`{scrollZoom (...) ['lasso2f'])` could also be a global variable for slightly better compression